packages/react/src/components/ChoiceGroup/ChoiceGroup.test.tsx

```diff
@@ -51,39 +37,39 @@ describe('ChoiceGroup', () => {
   });
 
   it('does not use className prop from parent on label', () => {
-    choiceGroup = mount(<ChoiceGroup className="testClassName" label="test label" options={TEST_OPTIONS} required />);
-    const label = choiceGroup.getDOMNode().querySelector('label');
+    const { container } = render(
+      <ChoiceGroup className="testClassName" label="test label" options={TEST_OPTIONS} required />,
+    );
+    const label = container.querySelector('label');
```

Comment:
Not explicitly with `getByRole("label")` since label doesnt have a corresponding aria role. I've changed this to use `getByRole("radiogroup").firstElementChild`